src/js/utils/media.js

```diff
@@ -1,4 +1,5 @@
 import loadImage from 'lib/load-image';
+const _ = require('underscore');
```

Comment:
Hmm I'm pretty sure we only use underscore in tests but not in the actual lib code. We do it that way to not add a dependency that could be replaced by vanilla js and reduce the bundle size.